### PR TITLE
Azure - simplify forms

### PR DESCRIPTION
**Pull request by jasmingoh**
After talking with our users and sponsors, we decided to remove the resource group field from both Cosmos DB and App Service forms, and remove the location field from the Cosmos DB form.

**Before**:
image
image
___
**After**:
image
image

___
**To test:**
- Create a project, add Cosmos DB and App Service
- After deployment is done, go to the portal and find your resource group
- Make sure that the Cosmos DB and App Service is in the resource group
- Both services should be in Central US

### CODE REVIEW

**Review by jsondoo — COMMENTED** [review]
We should also probably remove `chooseExistingRadioButtonSelected` from the initial states, React hooks and handlers (i.e. `radioButtonOnChangeHandler`) for the radio buttons.

File: src/client/src/containers/AppServiceModal/index.tsx [hunk 1]
```diff
@@ -259,7 +256,7 @@ const AppServiceModal = (props: Props) => {
       siteNameAvailability,
       setFormIsSendable
     );
-  }, [appServiceFormData.chooseExistingRadioButtonSelected]);
+  });
```
Comment (jsondoo) [inline src/client/src/containers/AppServiceModal/index.tsx]: we should remove this `useEffect` completely now

**Review by Tanya0609 — APPROVED** [review]
The form is much simpler now!

**Review by piotrmark — APPROVED** [review]

**Review by jcoc611-microsoft — APPROVED** [review]

**Review by SahilTara — APPROVED** [review]
I now understand what was being removed and why. Nice!

**Pull request merged**